<html>

<style>

body {
  background-color: #d0e4fe;
}

h1 {
  color: orange;
  text-align: center;
}

p {
  font-family: "Times New Roman";
  font-size: 20px;
}
</style>h

<body>

<img src="images.png" alt="" width="104" height="142">

<h1>
Bioinformatics

</h1>

<p>
Bioinformatics Listeni/?ba?.o???nf?r'mæt?ks/ is an interdisciplinary field that develops methods and software tools for understanding biological data. As an interdisciplinary field of science, bioinformatics combines computer science, statistics, mathematics, and engineering to study and process biological data.

</p>

<p>
Bioinformatics is both an umbrella term for the body of biological studies that use computer programming as part of their methodology, as well as a reference to specific analysis "pipelines" that are repeatedly used, particularly in the fields of genetics and genomics. Common uses of bioinformatics include the identification of candidate genes and nucleotides (SNPs). Often, such identification is made with the aim of better understanding the genetic basis of disease, unique adaptations, desirable properties (esp. in agricultural species), or differences between populations. In a less formal way, bioinformatics also tries to understand the organisational principles within nucleic acid and protein sequences.

</p>
<h1>
History
</h1>

<p>
Paulien Hogeweg and Ben Hesper coined the term bioinformatics in 1970 to refer to the study of information processes in biotic systems.[1][2][3] This definition placed bioinformatics as a field parallel to biophysics (the study of physical processes in biological systems) or biochemistry (the study of chemical processes in biological systems).[1]


</p>


<h1>
Sequences
</h1>
<p>
Computers became essential in molecular biology when protein sequences became available after Frederick Sanger determined the sequence of insulin in the early 1950s. Comparing multiple sequences manually turned out to be impractical. A pioneer in the field was Margaret Oakley Dayhoff, who has been hailed by David Lipman, director of the National Center for Biotechnology Information, as the "mother and father of bioinformatics."[4] Dayhoff compiled one of the first protein sequence databases, initially published as books[5] and pioneered methods of sequence alignment and molecular evolution.[6] Another early contributor to bioinformatics was Elvin A. Kabat, who pioneered biological sequence analysis in 1970 with his comprehensive volumes of antibody sequences released with Tai Te Wu between 1980 and 1991


</p>

<h1>
Genomes
</h1>
<p>

As whole genome sequences became available, again with the pioneering work of Frederick Sanger,[8] it became evident that computer-assisted analysis would be insightful. The first analysis of this type, which had important input from cryptologists at the National Security Agency, was applied to the nucleotide sequences of the bacteriophages MS2 and PhiX174. As a proof of principle, this work showed that standard methods of cryptology could reveal intrinsic features of the genetic code such as the codon length and the reading frame. This work seems to have been ahead of its time—it was rejected for publication by numerous standard journals and finally found a home in the Journal of Theoretical Biology.[9] The term bioinformatics was re-discovered and used to refer to the creation of databases such as GenBank in 1982. With public availability of data, tools for their analysis were quickly developed and described in journals, such as Nucleic Acids Research, which published specialized issues on bioinformatics tools as early as 1982.
</p>
<h1>
Goals
</h1>
<p>
To study how normal cellular activities are altered in different disease states, the biological data must be combined to form a comprehensive picture of these activities. Therefore, the field of bioinformatics has evolved such that the most pressing task now involves the analysis and interpretation of various types of data. This includes nucleotide and amino acid sequences, protein domains, and protein structures.[10] The actual process of analyzing and interpreting data is referred to as computational biology. Important sub-disciplines within bioinformatics and computational biology include:
<br>
Development and implementation of computer programs that enable efficient access to, use and management of, various types of information
Development of new algorithms (mathematical formulas) and statistical measures that assess relationships among members of large data sets. For example, there are methods to locate a gene within a sequence, to predict protein structure and/or function, and to cluster protein sequences into families of related sequences.
</p>
<p>
The primary goal of bioinformatics is to increase the understanding of biological processes. What sets it apart from other approaches, however, is its focus on developing and applying computationally intensive techniques to achieve this goal. Examples include: pattern recognition, data mining, machine learning algorithms, and visualization. Major research efforts in the field include sequence alignment, gene finding, genome assembly, drug design, drug discovery, protein structure alignment, protein structure prediction, prediction of gene expression and protein–protein interactions, genome-wide association studies, and the modeling of evolution.
</p>
<p>
Bioinformatics now entails the creation and advancement of databases, algorithms, computational and statistical techniques, and theory to solve formal and practical problems arising from the management and analysis of biological data.
</p>
<p>
Over the past few decades rapid developments in genomic and other molecular research technologies and developments in information technologies have combined to produce a tremendous amount of information related to molecular biology. Bioinformatics is the name given to these mathematical and computing approaches used to glean understanding of biological processes.
</p>
<h1>
Approaches
</h1>

<p>

Common activities in bioinformatics include mapping and analyzing DNA and protein sequences, aligning DNA and protein sequences to compare them, and creating and viewing 3-D models of protein structures.
</p>


<p>
There are two fundamental ways of modelling a Biological system (e.g., living cell) both coming under Bioinformatic approaches.
</p>

<p>
Static
<br>Sequences – Proteins, Nucleic acids and Peptides
<br>Interaction data among the above entities including microarray data and Networks of proteins, metabolites
Dynamic
<br>Structures – Proteins, Nucleic acids, Ligands (including metabolites and drugs) and Peptides (structures studied with bioinformatics tools are not considered static anymore and their dynamics is often the core of the structural studies)
<br>Systems Biology comes under this category including reaction fluxes and variable concentrations of metabolites
<br>Multi-Agent Based modelling approaches capturing cellular events such as signalling, transcription and reaction dynamics
<br>A broad sub-category under bioinformatics is structural bioinformatics.
</p>



</body>
</html>